tests/main/proxy/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: Ensure that the core.proxy.* settings are honored
+
+systems: [ubuntu-18.04-64]
```

Comment:
Is there any specific reason why this is scoped to only one release?

Reply:
No particular reason, mostly because I know the default squid install will work here. I was playing with creating a custom python based proxy but that is too much work with python to support CONNECT. I was also considering squid or a lightweight go proxy in a snap. I think that is the best approach - but I was lazy :/

Reply:
I think it is fine for testing it this way. We can expand on this in a subsequent branch. I definitely did not intend to block this PR on squid.